x-pack/plugins/apm/public/components/app/TransactionDetails/index.tsx

```diff
@@ -19,9 +19,10 @@ import { Transaction } from './Transaction';
 import { useLocation } from '../../../hooks/useLocation';
 import { useUrlParams } from '../../../hooks/useUrlParams';
 
-export function TransactionDetails() {
+export const TransactionDetails = () => {
```

Comment:
What's the reason of this change?

Reply:
Good question. I think I was experimenting with wrapping the component in `React.memo` and arrow function made that easier. I'll change it back.